docker-compose.yml

```diff
@@ -5,25 +5,25 @@
 version: '3.3'
 services: 
   webui:
-    image: ${WEBUI_IMAGE}    
+    image: intel/oact-webui:v2.1.1  
     networks:
       - openamtnetwork
-    build:
-      context: ./sample-web-ui
-      dockerfile: ./Dockerfile
+    # build:
+    #   context: ./sample-web-ui
+    #   dockerfile: ./Dockerfile
     environment:
       RPS_SERVER: https://${MPS_COMMON_NAME}/rps
       MPS_SERVER: https://${MPS_COMMON_NAME}/mps
       VAULT_SERVER: https://${MPS_COMMON_NAME}/vault
     volumes:
-      - ./sample-web-ui/nginx.conf:/etc/nginx/conf.d/default.conf  
+      - ./nginx.conf:/etc/nginx/conf.d/default.conf  
   rps:
-    image: ${RPS_IMAGE}
+    image: intel/oact-rps:v2.2.0
     networks:
       - openamtnetwork
-    build:
-      context: ./rps
-      dockerfile: ./Dockerfile
+    # build:
```

Comment:
we shouldn't remove these

Reply:
Can they stay uncommented without interfering? I'm assuming they only trigger when using '--build'? Because those dockerfile won't exist in the dockerhub path since we don't pull --recursive.

Reply:
thats correct